bindata/bootkube/scc-manifests/0000_20_kube-apiserver-operator_00_scc-anyuid.yaml

```diff
@@ -35,6 +35,7 @@ supplementalGroups:
 users: []
 volumes:
 - configMap
+- csi
```

Comment:
Discussed with @deads2k on slack, we need to enable `csi` volumes only when `CSIInlineVolumeAdmission` feature is enabled.

/lgtm cancel

1. We need two manifests with 0000_20_kube-apiserver-operator_00_scc-anyuid.yaml (and all other files).
    * One as it is, without `csi` volume, created when there is no feature set.
    * Second with `csi` volume, created only when TechPreviewNoUpgrade is set.
    See https://github.com/openshift/enhancements/blob/master/enhancements/update/cvo-techpreview-manifests.md#proposal how to annotate the manifests accordingly.
2. Check if the operator syncs manifests in the runtime. If so, sync the manifests with StaticResourceController with [conditional resources](https://github.com/openshift/library-go/blob/7f7c08fa194177a462d1065930cd1f756995e245/pkg/operator/staticresourcecontroller/static_resource_controller.go#L152) to always create the right version.

Reply:
I pushed a new commit to test, but I'll need to rebase once https://github.com/openshift/cluster-kube-apiserver-operator/pull/1380 merges.
So we now have 2 sets of manifests, one for techpreview, one for "everything else", and they both need the annotation. The only difference for each one is this:
```
22c22
<     release.openshift.io/feature-set: "Default,CustomNoUpgrade,LatencySensitive"
---
>     release.openshift.io/feature-set: "TechPreviewNoUpgrade"
38a39
> - csi
```
But I think / hope we can remove those annotations and consolidate back into a single set of manifests again once CSI inline volumes goes GA.

Reply:
Ok this is not good. They look as expected by default.
But if you:
1) `oc edit featuregate cluster` after install to set `featureSet: "TechPreviewNoUpgrade"`, or 
2) Set `featureSet: TechPreviewNoUpgrade` directly in install-config.yaml
In both cases, the new manifests do not get applied correctly.
Only the ones without `release.openshift.io/create-only` get the new `release.openshift.io/feature-set` value.
```
$ oc get scc -o yaml | grep feature-set
      release.openshift.io/feature-set: Default,CustomNoUpgrade,LatencySensitive
      release.openshift.io/feature-set: Default,CustomNoUpgrade,LatencySensitive
      release.openshift.io/feature-set: Default,CustomNoUpgrade,LatencySensitive
      release.openshift.io/feature-set: Default,CustomNoUpgrade,LatencySensitive
      release.openshift.io/feature-set: TechPreviewNoUpgrade
      release.openshift.io/feature-set: Default,CustomNoUpgrade,LatencySensitive
      release.openshift.io/feature-set: TechPreviewNoUpgrade
      release.openshift.io/feature-set: Default,CustomNoUpgrade,LatencySensitive
      release.openshift.io/feature-set: TechPreviewNoUpgrade
```
And none of them get the new `- csi` line under volumes.
```
$ oc get scc -o yaml | grep csi
$
```

Reply:
Since it appears there is currently no way to update SCC's when the TechPreviewNoUpgrade feature set is enabled, I've dropped these 2 commits for now. Linking them here to leave a record of what I tested:
https://github.com/openshift/cluster-kube-apiserver-operator/commit/ed66da27f9eedb58354fdc379071306f61cf06d4
https://github.com/openshift/cluster-kube-apiserver-operator/commit/51598da3569735fad68f25e6a66d0f8134b6d2e9
I'd like to use this PR to enable the admission plugin, even without the SCC changes.
When the feature goes GA, we can add `csi` to the SCC manifests, which will help with new installs.
But separately, we need some way to update SCC's after they're created: either automatically by an OCP operator, or a manual procedure that customers can follow. This affects other features as well, like generic ephemeral volumes which is already GA (https://github.com/openshift/cluster-kube-apiserver-operator/pull/1380).

Reply:
Ack, syncing SCCs during cluster upgrade / feature enablement is a separate issue, most likely to be solved by https://issues.redhat.com/browse/RFE-1977